dmv/dmv.js

```diff
@@ -19,13 +19,18 @@ DMV.prototype.currentCustomerFor = function(agent) {
 }
 
 DMV.prototype.nextCustomer = function() {
+  if (this.customers.length === 0) {
+    throw new Error('No customers')
+  }
   const currentCustomer = this.customers.shift()
```

Comment:
currentCustomer truly encapsulates the complexity of all this truly complex piece of code!

Reply:
Please provide another comment to demonstrate "Speak fluently using object-oriented / coding terminology; take code and describe it using words like method, parameter, declare, instantiate...etc"

Reply:
@quintanamiriam 
The thing that **does the encapsulation** or `encapsulates` is the containing function `.nextCustomer()` - or the `DMV class`/object. 

The things that **are encapsulated** are the `currentCustomer`, `this.customers`, `this.agentList`, etc